dev/utils/geoConverter.js

```diff
@@ -5,8 +5,11 @@ require('dotenv').config();
 
 const geoConverter = address =>
    axios.get(`${URL}${address}&key=${process.env.G_MAPKEY}`)
-    .then(response =>
-       response.data.results[0].geometry.location)
+    .then((response) => {
+      console.log('gmaps response', response.data);
+      // TODO: Fix this. Returning does nothing in a then block.
```

Comment:
@Chimpytheape not sure if it was already like this? Do you remember what you did with the data on line 11?